### ISSUE TITLE
Cipher not found cryptlib too old

### ISSUE THREAD

**Issue by gilojimjim** [klamonte/qodem#93]
Qodem doesn't works with BBS in SSH because cryptlib is too old and does not have new ciphers.

**Unable to negotiate with server port 22: no matching cipher found. The
ir offer: aes128-cbc,3des-cbc**

**you should use new cryptlib :**

https://cryptlib-release.s3-ap-southeast-1.amazonaws.com/cryptlib345.zip
https://www.cs.auckland.ac.nz/~pgut001/cryptlib/

**Comment by klamonte** [comment]
Thanks for reporting this.

Yeah, it's been that way for some time.  I assume you're using the Windows version?  For non-Windows systems, it's best to use the system-provided ssh.

Good to know CryptLib got updated.  I kept looking for like 2 years after OpenSSH deprecated the old ones and no dice.  I would have to retrofit some patches back to CryptLib (for reporting tty row/col size changes and some env vars if I remember).  If/when I do another release I will look at a CryptLib update then too.
